src/navsections/HelmChartSectionBlueprint/HelmChartContextMenu.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import {Menu} from 'antd';
+
+import styled from 'styled-components';
+
+import {ItemCustomComponentProps} from '@models/navigator';
+
+import {useAppDispatch, useAppSelector} from '@redux/hooks';
+import {selectFile, setSelectingFile} from '@redux/reducers/main';
+import {setLeftMenuSelection} from '@redux/reducers/ui';
+import {isInPreviewModeSelector} from '@redux/selectors';
+
+import {Dots} from '@atoms';
+
+import ContextMenu from '@components/molecules/ContextMenu';
+
+import Colors from '@styles/Colors';
+
+const StyledActionsMenuIconContainer = styled.span<{isSelected: boolean}>`
```

Comment:
Can we have the styled components in a separate file `.styled` ?

Reply:
done